src/app/shared/components/VueNavBar/VueNavBar.spec.ts

```diff
@@ -80,4 +80,27 @@ describe('VueNavBar.vue', () => {
     wrapper.update();
     expect(wrapper.vm.isOpen).toBeFalsy();
   });
+
+  test('should close menu when item is clicked', () => {
+    const wrapper: any = mount(VueNavBar, {
```

Comment:
you could write the test like this:

- initialize the wrapper with isOpen
- use the global event bus to trigger the close event
- assert if wrapper.vm.isOpen is false

this would be independent of the child component.

if you want to do it more like an integration test, you have to provide a slot with some markup or a test component which triggers the event bus and then find the right html element to trigger the click event. 

the navbar itself has no links in it (as you see in App.vue), it just provides a slot for a child component.